Support save pod/container std log on the Environment (#62)
apache/skywalking-infra-e2e: docs/en/setup/Configuration-File.md

      url: http://${pod_foo_host}:${pod_foo_8080}/
   ```

#### Log

> The console output of each pod will be saved after startup. File address: `${workDir}/logs/${namespace}/${podName}.log`

Reviewer: ```suggestion
The console output of each pod could be found in `${workDir}/logs/${namespace}/${podName}.log`
```

      url: http://${oap_host}:${oap_8080}/
   ```

#### Log

> The console output of each service will be saved after startup. File address: `${workDir}/logs/{serviceName}/std.log`

Reviewer: ```suggestion
The console output of each service could be found in `${workDir}/logs/{serviceName}/std.log`
```
